frontend/console/src/features/timeline/TimelineIcon.tsx

```diff
@@ -2,7 +2,6 @@ import {
   Call02Icon,
   CallIncoming04Icon,
   ComputerIcon,
-  CustomerServiceIcon,
```

Comment:
@wesbillman Does anything else need to be removed now that we don't used this icon anywhere else?